src/test/java/org/mifos/integrationtest/cucumber/stepdef/BatchApiStepDef.java

```diff
@@ -481,7 +481,7 @@ public void operationsBatchTestTearDown() {
 
     @When("I call the batch aggregate API with expected status of {int} with total {int} txns")
     public void iCallTheBatchAggregateAPIWithExpectedStatusOf(int expectedStatus, int totalTxns) {
-        await().atMost(awaitMost, SECONDS).pollDelay(pollDelay, SECONDS).pollInterval(pollInterval, SECONDS).untilAsserted(() -> {
+        await().atMost(awaitMost, SECONDS).pollDelay(5, SECONDS).pollInterval(pollInterval, SECONDS).untilAsserted(() -> {
```

Comment:
do we need to hard code the poll delay?

Reply:
@fynmanoj  Otherwise BD-013 is failing so.